lib/pure/collections/sequtils.nim

```diff
@@ -509,12 +509,37 @@ proc keepIf*[T](s: var seq[T], pred: proc(x: T): bool {.closure.})
       inc(pos)
   setLen(s, pos)
 
-func delete*[T](s: var seq[T]; first, last: Natural) =
+func delete*[T](s: var seq[T]; slice: Slice[int]) =
+  ## Deletes the items `s[slice]`, raising a defect if the slice contains
+  ## elements out of range.
```

Comment:
```suggestion
  ## Deletes the items `s[slice]`, raising a defect if the slice contains
  ## elements out of range.
  ##
  ## This operation moves all elements after `s[slice]` in linear time.
```
Wording can be changed, but it should be clear that this is an O(n) operation.

Reply:
thanks, done